site/layouts/shortcodes/orange-footer.html

```diff
@@ -39,7 +39,7 @@ <h3 class="footer-heading">Sign up to our mailing list</h3>
 {{- end }}
 {{- if $follow }}
   <div class="container-xxl footer-social">
-    <h3 class="footer-heading">Follow us</h3>
+    <h3 class="footer-heading me-md-3">Follow us</h3>
```

Comment:
Could be `.me-3`

Reply:
`.me-md-3` is more specific so sounds better to me